### PR TITLE
Refactor docker compose env

### PR DESCRIPTION
**Pull request by hjmtql**
Since the test code now refers to the `.env` variables,
I rewrote the hard-coded part of `docker-compose.yml`.

### CODE REVIEW

File: .env-ci [hunk 1]
```diff
@@ -1,5 +1,5 @@
-PG_DB="purspg"
```
Comment (hjmtql) [inline .env-ci:1]: There was a problem with double quotes in docker-compose, so I deleted it.
This change seems to be fine for reading dotenv in purescript code.

↓The problem before change
```
$ docker-compose config
services:
  db:
    environment:
      POSTGRES_DB: '"purspg"'
      POSTGRES_PASSWORD: '"qwerty"'
      POSTGRES_USER: '"init"'
    image: postgres:12.5
    ports:
    - 5432:5432/tcp
version: '3.0'
```

After
```
$ docker-compose config
services:
  db:
    environment:
      POSTGRES_DB: purspg
      POSTGRES_PASSWORD: qwerty
      POSTGRES_USER: init
    image: postgres:12.5
    ports:
    - 5432:5432/tcp
version: '3.0'
```

File: package.json [hunk 2]
```diff
@@ -8,7 +8,6 @@
   },
   "scripts": {
     "lit": "rm -rf guide/src/ && paluh-litps --input guide/ --output guide/src/ && cp guide/Custom.js guide/src/Custom.js",
-    "env": "[ -f .env-ci ] && export $(cat .env-ci | xargs)",
```
Comment (hjmtql) [inline package.json:11]: I removed it because it's seemed to be no longer used.

File: docker-compose.yml [hunk 3]
```diff
@@ -4,8 +4,8 @@ services:
   db:
     image: postgres:12.5
     environment:
-      POSTGRES_DB: purspg
```
Comment (hjmtql) [inline docker-compose.yml:7]: NOTICE: `docker-compose` command reads `.env` file's variables by defalut.

**Comment by Kamirus** [comment]
@hjmtql Thank you!

**Pull request merged**